src/index.html

```diff
@@ -8,6 +8,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <!-- <h1>Stars</h1> -->
```

Comment:
remove commented code
with git history there is no need for commented code.
and in this case, it does not bring anything useful